l10n_es_facturae_face/tests/test_facturae_face.py

```diff
@@ -174,6 +176,11 @@ def setUpClass(cls):
             "l10n_es_facturae_face.facturae_face_update_exchange_type"
         )
 
+    # Don't block external requests.
+    @classmethod
+    def _request_handler(cls, s, r, /, **kw):
+        return cls._super_send(s, r, **kw)
+
```

Comment:
I think this is not necessary either

Reply:
It is necessary so external calls don't get blocked.